### PR TITLE
Django 2.1 support

### PR DESCRIPTION
**Pull request by tpeaton**
This PR renames the `Visitor` permissions originally called `view_visitor` which clashes with the permission Django 2.1 auto-generates.  Unfortunately, this requires users to apply the new migration, but that should only be a mild inconvenience.  I've also dropped support for older pythons/djangos and made the build matrix consistent with what Django currently supports.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,45 +4,35 @@ sudo: false
...
 env:
-    - DJANGO="Django>=1.8,<1.9"
-    - DJANGO="Django>=1.9,<1.10"
-    - DJANGO="Django>=1.10,<1.11"
     - DJANGO="Django>=1.11,<2.0"
     - DJANGO="Django>=2.0,<2.1"
+    - DJANGO="Django>=2.1,<2.2"
 
 matrix:
     exclude:
```
Comment (mjschultz) [inline .travis.yml]: I think the exclusion matrix should _just_ be:
```
        - python: "2.7"
          env: DJANGO="Django>=2.0,<2.1"
        - python: "2.7"
          env: DJANGO="Django>=2.1,<2.2"
        - python: "3.4"
          env: DJANGO="Django>=2.1,<2.2"
```

for now.

**Comment by jdemaeyer** [comment]
We've been using this in a (low-traffic) production environment for a while and it works like a charm, any chance it's gonna land in a release soon? :)

**Comment by robmoggach** [comment]
commit this please....

**Comment by bruth** [comment]
0.5.0 released

**Pull request merged**